templates/registration/password_reset_complete.html

```diff
@@ -6,9 +6,11 @@
 
 {% block content %}
 
-<div class="container">
-    <div class="d-flex flex-column">
-        <p class="m-auto">Your password has been reset. You may go ahead and <a href="{% url 'accounts:login_signup' %}">sign in</a> now.</p>
+<div class="container" style="margin-top: 150px;">
+    <div class="d-flex flex-column text-center" style="margin: auto; max-width: 500px; align-items: center;">
+        <i class="fas fa-check-circle my-3" style="color: #28a745; font-size: 7em;"></i>
```

Comment:
You can use the tick.gif present in `static/accounts/icon` directory of the project instead of this icon.

Reply:
I'm getting this error when I try to, could you please help me out? 

![Screenshot (96)](https://user-images.githubusercontent.com/64403591/112148298-b5ee0e80-8c03-11eb-8119-d74919630504.png)

Reply:
oh sorry I forgot to add load static at top. I fixed it